lab11_2/src/main.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { useState } from 'react';
+
+import './style.css';
+
+const App = () => {
+  const [time, setTime] = useState(0);
+  return (
+    <div id="app-container">
+      <label>Hours:</label>
+      <input value={unConvert(time, 'h')} onChange={e => setTime(converter(e.target.value, 'h'))}/>
+      <br></br>
+      <label>Minutes:</label>
+      <input value={unConvert(time, 'm')} onChange={e => setTime(converter(e.target.value, 'm'))}/>
+      <br></br>
+      <label>Seconds:</label>
+      <input value={unConvert(time, 's')} onChange={e => setTime(converter(e.target.value, 's'))}/>
+      <br></br>
+    </div>
+  );
+
+  function unConvert(val, identifier) {

```

Comment:
There is no need to skimp on the length of the variable names, it is a horrible legacy from times when a few extra characters made a big difference. Also, one-letter variables are absolutely not allowed in this course:)
`unitConvert(value, identifier)` or 
```js
hours: value / 60 / 60,
minutes: value / 60,
seconds: value
```
would be much clearer.

Reply:
Otherwise, this is a good solution:)